index.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html lang="en" >
+<head>
+<title>Tic-tac-toe-js</title>
+<!-- Required meta tags -->
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<!-- Bootstrap CSS -->
+<link rel="stylesheet" href="bootstrap/css/bootstrap.min.css" >
+
+<!--font google-->
+<link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+
+<!--main style-->
+<link rel="stylesheet" href="css/style.css">
+
+</head>
+<body>
+
+
+<!--main-header START-->
+<header id="main-header">
+   <h1>Tic Tac Toe</h1>
+</header>
+<!--main-header STOP-->
+
+
+
+
+<!--section GAME START-->
+<section id="game-section">
+    <div class="container game-container">
+        <span id="player">PLAY</span>
+        <div class="big-box" id="big-box">
+            <div class="small-box" id="box1"></div>
+            <div class="small-box" id="box2"></div>
+            <div class="small-box" id="box3"></div>
+            <div class="small-box" id="box4"></div>
+            <div class="small-box" id="box5"></div>
+            <div class="small-box" id="box6"></div>
+            <div class="small-box" id="box7"></div>
+            <div class="small-box" id="box8"></div>
+            <div class="small-box" id="box9"></div>
+        </div>
+        <div class="text-center">
+                <button type="button" id="play-again" class="btn btn-success">Play Again</button>
+        </div>
+        <div class="clear"></div>
+    </div>
+</section>
+<!--section GAME STOP-->
+
+
+
+<!--FOOTER START-->
+
+
+<footer id="footer">
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-6">
+                    <p>Tic Tac Toe GAME</p>
+            </div>
+            <div class="col-sm-6">
+                  <p>Mateusz Łubianka</p>  
+            </div>
+        </div>
+    </div>
+</footer>
+
+
+<!--FOOTER STOP-->
+
+
+
+<script src="js/jquery-3.3.1.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js"></script>
+<script src="bootstrap/js/bootstrap.min.js"></script>
+<script src="js/main.js"></script>
+</body>
+</html>
```

Comment:
na końcu pliku powinien się znaleźć pusty wers. Możesz to zautomatyzować podczas zapisu pliku https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
na końcu pliku nie po body. Nie ważna jaki plik  css, js, html czy jakikolwiek inny, na końcu powinien być pusty wers